sandbox/public/e2e/js/__tcfapi.js

```diff
@@ -0,0 +1,119 @@
+(function(global, factory) {
```

Comment:
Did you write this or was it auto-generated?

Reply:
It was auto-generated. I don't think we need that wrapper right?